docs/upstream-integrations/new-bot-command/slash-cmd-job.md

```diff
@@ -6,11 +6,15 @@ sidebar_position: 9
 
 # Creating the Hello Goodbye Slash Command Job
 
+Diagram below shows the file structure and thier requiremnt.
+
 ![Screen Shot 2022-06-16 at 2.51.58 PM.png](imgs/img20.png)
 
+The diagram below shows Slash Command with intermediate response and follow up after executing a job, this is the final overview of this walkthrough.
+
 ![Screen Shot 2022-06-16 at 11.08.11 AM.png](imgs/img21.png)
 
-## Step 1: Understand why we need a Job Server
+## Step 1: Understand why we need a Job Server?
```

Comment:
I might change this to 
"ing" remove the "?"

"Understanding why we need a Job Server"